pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1415,6 +1416,10 @@ def __init__(
             ).encode()
 
         if dask_client is not None:
+            self.config_options["PROTOCOL".encode()] = parse_address(
```

Comment:
We want to be able to have BSQL using TCP and Dask with UCX and vice-versa. This is something we are actually going to be testing very soon to make sure it works, so we dont want it to always just use whatever dask is using. I suggest we have PROTOCOL default to "AUTO", so if the user does not explicitly set it, it will be set by whatever dask is using. BUT if a user goes out of their way to set it, it should use what the user set.